sklearn/decomposition/_kernel_pca.py

```diff
@@ -69,16 +97,25 @@ class KernelPCA(TransformerMixin, BaseEstimator):
         Maximum number of iterations for arpack.
         If None, optimal value will be chosen by arpack.
 
+    iterated_power : int >= 0, or 'auto', (default 'auto')
+        Number of iterations for the power method computed by
+        svd_solver == 'randomized'.
+
+        .. versionadded:: 0.23
+
     remove_zero_eig : boolean, default=False
         If True, then all components with zero eigenvalues are removed, so
         that the number of components in the output may be < n_components
         (and sometimes even zero due to numerical instability).
         When n_components is None, this parameter is ignored and components
         with zero eigenvalues are removed regardless.
 
-    random_state : int, RandomState instance, default=None
-        Used when ``eigen_solver`` == 'arpack'. Pass an int for reproducible
-        results across multiple function calls.
+    random_state : int, RandomState instance or None, optional (default=None)
```

Comment:
please revert. The previous form was correct

Reply:
ok. I'll also fix n_jobs that was in the other form